### DISCUSSION TITLE
Notification Samples

### DISCUSSION THREAD

**Discussion by csharpcoder198** [shinyorg/shiny#1027] [Ideas]
Hello:
   I need to compile / run the Notification Samples. I had issues with the Geofencing sample previously. If you recall you tweaked the code and I was compiling against the released version.

I was unclear about how to get the Library upgraded from preview. Not as straight forward as updating the Nugets or even the Build.props/targets file. Or is it?

Oh same request for Jobs Samples. Thanks!

Thanks

**Comment by aritchie** [comment]
I have updated the project to use the latest version.

To update the nuget for most of the samples, you can do it under Directory.build.props.  There is a field called ShinyVersion.
